config/initializers/track_search.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Rails.application.config.after_initialize do
```

Comment:
this looks like a migration which it is.
These search tables are in memory only so if you make a proper db migration then it doesn't know how to dump that to schema so it doesn't really work.

this is weird but so is using sqlite3 in prod.